### PR TITLE
Assessment

### PR DESCRIPTION
**Pull request by ryoconnell**

### CODE REVIEW

File: challenge.md [hunk 1]
```diff
@@ -6,37 +6,37 @@ Write your answers inside this file, where it's indicated by the comments.
...
 
-<!-- Answer Starts Here -->
+<!-- Answer Starts Here -->git clone url
 
 <!-- Answer Ends Here -->
 
 2. OK, we just wrote some code. What command could we use to see a list of all the changes that have been made since the last commit?
 
-<!-- Answer Starts Here -->
+<!-- Answer Starts Here -->git status
```
Comment (GA-MEB) [inline challenge.md:15]: Yep. But `git status` will only tell us _which files_ have changed; we'd also probably want to use `git diff` to show what the changes themselves were.

File: challenge.md [hunk 2]
```diff
@@ -6,37 +6,37 @@ Write your answers inside this file, where it's indicated by the comments.
...
 
-<!-- Answer Starts Here -->
+<!-- Answer Starts Here -->git commit --amend
 
 <!-- Answer Ends Here -->
 
 4. It looks like there was a change on the original repo that we forked from. How could we grab those changes and incorporate them into our local repo?
 
-<!-- Answer Starts Here -->
+<!-- Answer Starts Here -->origin...not sure.
```
Comment (GA-MEB) [inline challenge.md:27]: `origin` refers to our fork - the repo that our fork was copied from is usually referred to as `upstream`. And to grab changes from someplace else and incorporate them into our current project, the command is `git pull`. 

So, tying it all together, you'd need to type `git pull upstream` (plus, optionally, a branch, such as `master`).

File: challenge.md [hunk 3]
```diff
@@ -6,37 +6,37 @@ Write your answers inside this file, where it's indicated by the comments.
...
 
-<!-- Answer Starts Here -->
+<!-- Answer Starts Here -->git reset --hard HEAD~3
 
 <!-- Answer Ends Here -->
 
 6. OK, we've done all the work we want to do locally. Let's update our fork so that we can make a pull request. What command would we use to update our fork?
 
-<!-- Answer Starts Here -->
+<!-- Answer Starts Here -->git push, pull request on github.
```
Comment (GA-MEB) [inline challenge.md:39]: Specifically, we'd need to do `git push origin` (optionally with `master` as the specific branch).

File: challenge.md [hunk 4]
```diff
@@ -6,37 +6,37 @@ Write your answers inside this file, where it's indicated by the comments.
...
 
-<!-- Answer Starts Here -->
+<!-- Answer Starts Here -->origin...not sure.
 
 <!-- Answer Ends Here -->
 
 5. Suppose that we wanted to look at the third-to-last commit on `master`; what command(s) would we write to do that?
 
-<!-- Answer Starts Here -->
+<!-- Answer Starts Here -->git reset --hard HEAD~3
```
Comment (GA-MEB) [inline challenge.md:33]: Oh no - please don't do that! `git reset --hard` resets both Git and your actual files to a prior state, **permanently**. 

If we just want to look at a previous commit, we can use `git checkout`.

Also, the third-to-last commit is `HEAD~2`, not `HEAD~3`.

**Comment by GA-MEB** [comment]
3 / 6

Keep practicing this stuff, Ryan - it's going to be super-important to be fluent with Git and GitHub as you go through the course. If you have time, I recommend coming back to this assessment in a few days and trying it again; I'd be happy to re-grade your work.

**Comment by ryoconnell** [comment]
Thanks.  I'll take it again this week.  I'm studying the steps now, and
based on the amount we are doing, I'd be surprised if I didn't have it down
by tomorrow afternoon.

**Pull request closed without merging**